test/test-applications/security/aspnet/Samples.AspNetMvc5/Views/Web.config

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0"?>
```

Comment:
This file seems misplaced, and is this supposed to replace the `Web.config` at the root of the application? Is the other one a symlink or something?

Reply:
It's part of the default template, other .NET Framework samples have a web.config in this directory.